src/services/institution-service.js

```diff
@@ -1,20 +1,16 @@
 import sequelizeConn from '../util/db.js'
 
 async function fetchInstitutions(projectId) {
-  let [rows] = await sequelizeConn.query(
+  const [rows] = await sequelizeConn.query(
     `
-      SELECT ins.name as name
+      SELECT ins.name as name, ins.institution_id as institutionId
```

Comment:
can you capitalize the SQL keywords "AS" ? Also, you don't need the renaming here so you can remove it.